packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -969,6 +998,9 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
     if (newViewProps.nextFocusRight.has_value()) {
       UIView *rootView = [self containingRootView];
       _nextFocusRight = [rootView viewWithTag:newViewProps.nextFocusRight.value()];
+      if (self.isFocused) {
+        [self enableNextFocusRight];
+      }
```

Comment:
I'd just call the `enableDirectionalFocusGuides` wherever I need it combined with the way I described above. So I wouldn't create separate `enableNext*` functions. You'd make this PR quite a bit shorter that way.